iot_config/migrations/20241203190903_solana_net_ids.sql

```diff
@@ -0,0 +1,9 @@
+-- Migration here solely for testing purposes
+-- An instance of account-postgres-sink
+-- Will alter this table depending on an on-chain struct
+CREATE TABLE IF NOT EXISTS solana_net_ids (
+    address TEXT PRIMARY KEY,
+    id INTEGER NOT NULL,
+    authority TEXT NOT NULL,
```

Comment:
Who would be authority?

Reply:
We would be the authority due to backfilling them with a wallet. We can also create additional net_ids with any wallet as the authority.